frontend/smart-kickers-game/src/components/Game/GameStatistics/StatisticsItems/FastestShot.js

```diff
@@ -0,0 +1,19 @@
+import React from 'react';
+import './StatisticItem.css';
+
+function FastestShot({ blue, white, statistics }) {
+  function returnFastestShot(teamID) {
+    if (!statistics?.fastestShot) return '😞';
+    const { speed, team } = statistics.fastestShot;
```

Comment:
I'm not sure about this - if I start a game and finish it (without actually playing), there is `fastestShot` defined, but as `fastestShot: {speed: 0, team: 0}`. Shouldn't it be just `null`/`undefined` if there was not shots at all?

Then we may not render this component at all, since there was no shots.

Reply:
It has been fixed. The component won't be rendered now.